en/stacks/index.md

```diff
@@ -0,0 +1,38 @@
+---
+title: What is Ubuntu Stacks?
+---
+
+# What is Ubuntu Core Stacks?
+
+Ubuntu Core Stacks is a set of snap packages that offer system-level functionality that
+enables important core system-level features not offered by the base Ubuntu Core
+system. With little to no additional effort, these snaps can be dropped in place
+on one or more new or existing systems bringing significant new value to these systems.
+These snaps have been well tested in various commercial offerings and are
+actively maintained by an internal Canonical team.
+
+Please note that some additional integration work to these snaps are required if
+new application of these snaps diverges significantly from existing applications.
```

Comment:
What do you mean with this paragraph? I have the feeling that the word 'application' is a bit overloaded in this context when talking about snaps as it has a different meaning there.